index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&amp;family=Poppins&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="nav">
+        <img class="geom" src="./images/image_geometry_1.svg" alt="geometry">
+        <img class="hamb" src="./images/ic_menu.svg" alt="hamburguer">
+    </header>
```

Comment:
Kindly notice that the [header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) element should contain a [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)  with your navigation links (in this case, your hamburger icon). Please move your hamburger icon to inside of a `nav` element 😀.

You can have something like this: 
```html
<header>
<img class="geom" ... >

  <nav>
    Hamburger icon
  </nav>
</header>
```